lib/internal/bootstrap/node.js

```diff
@@ -231,6 +231,10 @@
 
     setupAllowedFlags();
 
+    if (process.versions.openssl) {
+      internalBinding('crypto').loadExtraRootCertsFile();
+    }
```

Comment:
Could this possibly be injected via bootstrapper.cc instead? Especially since it's only used here.

Reply:
Now, it's removed to UseExtraCaCerts.